src/components/WorkPoolQueueStatusArray.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="work-pool-queue-status-array">
+    <template v-if="!showTooMany && workPoolQueues.length > 0">
+      <WorkPoolQueueStatusIcon
+        v-for="workQueue in workPoolQueues"
+        :key="workQueue.id"
+        :work-queue-name="workQueue.name"
+        :work-pool-name="workPool.name"
+        class="work-pool-queue-status-badge__icon"
+      />
+    </template>
+    <span v-if="!showTooMany && workPoolQueues.length < 1" class="work-pool-queue-status-array__none">N/A</span>
+    <span v-if="showTooMany">Too many to show here.</span>
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { useSubscription } from '@prefecthq/vue-compositions'
+  import { computed } from 'vue'
+  import { WorkPoolQueueStatusIcon } from '@/components'
+  import { useWorkspaceApi } from '@/compositions'
+  import { WorkPool } from '@/models'
+
+  const props = defineProps<{
+    workPool: WorkPool,
+  }>()
+
+  const maxWorkQueues = 50
+  const api = useWorkspaceApi()
+
+  const workPoolQueuesSubscription = useSubscription(
+    api.workPoolQueues.getWorkPoolQueues,
+    [props.workPool.name, { limit: maxWorkQueues }],
+  )
+  const workPoolQueues = computed(() => workPoolQueuesSubscription.response ?? [])
+  const showTooMany = computed(() => workPoolQueues.value.length >= maxWorkQueues)
```

Comment:
so since you used the limit on the request itself you can get rid of this because you'll never get more than 50 back from the api